inc/class-importer.php

```diff
@@ -313,9 +322,24 @@ protected function update_post_from_markdown_source( $post_id ) {
 		}
 		wp_update_post( $post_data );
 
+		// Add meta data from YAML front matter.
+		if ( isset( $yaml['meta'] ) && is_array( $yaml['meta'] ) ) {
+			foreach ( $yaml['meta'] as $key => $value ) {
+				update_post_meta( $post_id, $key, $value );
```

Comment:
`$key` and `$value` both need to be slashed. Also, might be worth using a filterable whitelist of keys here.